ccmlib/scylla_node.py

```diff
@@ -584,28 +590,41 @@ def import_bin_files(self):
         os.makedirs(os.path.join(self.get_path(), 'resources', 'cassandra',
                                  'tools', 'bin'))
         for name in files:
-            self.hard_link_or_copy(os.path.join(self.get_install_dir(),
-                                                'resources', 'cassandra',
+            self.hard_link_or_copy(os.path.join(self.get_tool_java_dir(),
                                                 'tools', 'bin', name),
                                    os.path.join(self.get_path(),
                                                 'resources', 'cassandra',
                                                 'tools', 'bin', name))
 
         # TODO: - currently no scripts only executable - copying exec
         scylla_mode = self.cluster.get_scylla_mode()
-        self.hard_link_or_copy(os.path.join(self.get_install_dir(),
-                                            'build', scylla_mode, 'scylla'),
-                               os.path.join(self.get_bin_dir(), 'scylla'))
-        self.hard_link_or_copy(os.path.join(self.get_install_dir(),
-                                            '..', 'scylla-jmx', 'target',
-                                            'scylla-jmx-1.0.jar'),
-                               os.path.join(self.get_bin_dir(),
-                                            'scylla-jmx-1.0.jar'))
-        self.hard_link_or_copy(os.path.join(self.get_install_dir(),
-                                            '..', 'scylla-jmx', 'scripts',
-                                            'scylla-jmx'),
-                               os.path.join(self.get_bin_dir(),
-                                            'scylla-jmx'))
+        if scylla_mode == 'reloc':
+            relative_repos_root = '../..'
+            self.hard_link_or_copy(os.path.join(self.get_install_dir(), 'bin', 'scylla'),
+                                   os.path.join(self.get_bin_dir(), 'scylla'))
+
+            os.symlink(os.path.join(self.get_install_dir(), 'libexec'),
```

Comment:
why is this a symlink and not a hard_lonk_or_copy like the others ?

Reply:
since hard_link_or_copy doesn't handle directories correctly.
the fallback shutil.copy isn't recursive

Reply:
@fruch the issue with symlink is that after you create a cluster with CCM its supposed to remain using the version you created it with not change under your "feet" when a new relocatable package is used.

Thats why we used hard_link or copy so if its on the same volume a hard link sufficies and if its not we copy the executable don't soft link to them.

Lets fix this - it caused issues in the past

- if the issue is only a symlink to a repo then I can leave with it (given you create a followup issue to fix it)
- if the issue is a symlink to a local relocatable that can change - thats not fine and we need a fix.

Reply:
I can see why it would be a problem when working from the build directory of scylla,
in relocatable is a bit likely to change under your feet, but I'll change it to hardlink / cp all the files in those two directories